org-asterion-dev/README.org

```diff
@@ -0,0 +1,69 @@
+#+TITLE: Asterion infrastructure deployment
+#+AUTHOR: James Blair, Shawn Gerrard, Daljit Singh
+#+DATE: <2022-03-12 Sat>
+
+
+This repository captures the end to end workflow for deploying asterion digital aws organization and iam via [[https://www.pulumi.com/][pulumi]].
+
+Please note that this stack requires the [[https://github.com/asterion-digital/asterion-as-code/tree/master/org-asterion][org-asterion]] pulumi stack brought `up`.
+
+
+* Setup environment
+
+To get started lets initialise our project, making sure we are in the right directory, have the python [[https://docs.python.org/3/library/venv.html][virtual environment]] activated, and have installed our python dependencies with [[https://pypi.org/project/pip/][pip]].
+
+#+NAME: Setup environment
+#+begin_src tmate
+# Start from the infra directory and initialise
+cd org-asterion-dev && pulumi stack init dev
```

Comment:
Is it necessary to have a separate directory for `org-asterion-dev`? 

Could we just use `org-asterion` and then have multuple `Pulumi.dev.yaml`, `Pulimi.test.yaml` etc files in the one directory?

Reply:
Makes sense - I think this can be done but I'll need to look into it.

Could I include any updates on this as another PR and merge in the current changes?

Reply:
Yeah that sounds good.  That's a great summary link! the approach shown here looks like it would cut down on complexity and duplication: https://www.pulumi.com/docs/guides/organizing-projects-stacks/#monorepo-with-base-infrastructure-project 

i.e:
```text
├─ org-asterion
  ├── index.ts
  ├── Pulumi.yaml
  ├── Pulumi.dev.yaml
  ├── Pulumi.staging.yaml
  └── Pulumi.prod.yaml
```